src/snowcli/config.py

```diff
@@ -136,6 +136,9 @@ def _dump_config(self):
         with open(self.file_path, "w+") as fh:
             dump(self.conf_file_cache, fh)
 
+    def add_temporary_connection(self):
```

Comment:
I can't find any usages of this method.

Reply:
It was an artifact of a different approach - deleted